docs/releases/updating_ez_platform.md

```diff
@@ -504,6 +504,12 @@ Some versions require updates to the database. Look through [the list of databas
         ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
         ```
 
+    !!! caution "Form (ezform) Field Type"
+    
+        In eZ Platform 2.3 forms are content like everything else. After the update, in order to create forms, you have to 
```

Comment:
Can we get rid of the first sentence? This page is complex enough with all the instructions, I don't think we need an explanation here.